api/definitions.yaml

```diff
@@ -153,6 +153,11 @@ ExtendedProfile:
       $ref: "#/EmailAddress"
     preferred_languages:
       $ref: "#/PreferredLanguages"
+    is_storage_of_message_content_enabled:
```

Comment:
I would use a shorter name, like `inbox_enabled`, "whether the user enabled the inbox feature, for details on how the inbox feature works see..."